lib/apiservers/portlayer/restapi/handlers/containers_handlers.go

```diff
@@ -261,7 +264,8 @@ func (handler *ContainersHandlersImpl) GetContainerListHandler(params containers
 		info.ContainerID = &container.ExecConfig.ID
 		info.LayerID = &container.ExecConfig.LayerID
 		info.Created = &container.ExecConfig.Created
-		info.Status = &container.Status
+		info.Status = new(string)
+		*info.Status = container.State.String()
```

Comment:
@hickeng Don't think this needs to change, it is consistent with our existing pattern where we deal with 'string *' fields.  And `... = &container.State.String()` won't compile.